modules/serverless-ossm-setup.adoc

```diff
@@ -183,6 +184,147 @@ spec:
 $ oc apply -f <filename>
 ----
 
+. Install Knative Eventing by creating the following `KnativeEventing` custom resource definition (CRD), which also enables the Istio integration:
++
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec:
+  config:
+    features:
+      istio: enabled
```

Comment:
is `config.features.istio` in KnativeEventing a thing now? (my impression from the design doc was that this idea was rejected)

Reply:
Yes, we need this for https://issues.redhat.com/browse/SRVKE-1446 to enable/disable the controller, at the design doc time SRVKE-1446 wasn't explored, feel free to propose alternatives